### PR TITLE
Move up wiki GitHub button on mobile

### LINKED ISSUE #4215: GitHub button on the osu! wiki blocks title on mobile
screenshot_20190227-101046

### PR DESCRIPTION
**Pull request by gary-kim**
Closes #4215

### CODE REVIEW

**Comment by gary-kim** [comment]
I had not thought of that. Sounds like a good idea.

**Comment by notbakaneko** [comment]
the forum uses the same class for each post as well, so the element style should probably have a wiki-specific `--wiki` modifier as we currently do not want the buttons on the forum post to be vertical

**Comment by gary-kim** [comment]
Sorry, I didn't see that change. This should now modify it in the wiki only.
Let me know if you'd like me to squash the commits.

File: resources/assets/less/bem/osu-page-header.less [hunk 1]
```diff
@@ -301,6 +311,10 @@
     position: absolute;
     bottom: 10px;
 
+    @media @mobile {
```
Comment (notbakaneko) [inline resources/assets/less/bem/osu-page-header.less]: This should be scoped only to the wiki's version of the header `.@{top}--wiki`

File: resources/assets/less/bem/osu-page-header.less [hunk 2]
```diff
@@ -156,6 +156,16 @@
     .@{top}:hover & {
       opacity: 1;
     }
+
+    & .forum-post-actions {
```
Comment (notbakaneko) [inline resources/assets/less/bem/osu-page-header.less]: this should be defined in `forum-post-actions.less` instead and scoped to just the wiki

**Comment by gary-kim** [comment]
The last commit has the requested changes.

**Pull request merged**